webpack.config.babel.js

```diff
@@ -37,12 +37,17 @@ const productionPlugins = [
     // this is actually the property name https://github.com/kimhou/replace-bundle-webpack-plugin/issues/1
     partten: /throw\s+(new\s+)?[a-zA-Z]+Error\s*\(/g,
     replacement: () => 'return;('
-  }])
+  }]),
+
+  new CopyWebpackPlugin([
+    { from: './dropdown-arrow-down.png', to: 'dropdown-arrow-down.png' }
```

Comment:
Polluting the markup is not an issue I've encountered before. I think it's more of a thing where 40 layers of nested divs "feel" wrong, but they really have no real usability impacts on AT.

That being said you can [use SVG as a background image](https://codepen.io/netsi1964/pen/HGJms). And if the source to it is inlined as in the CodePen example I just linked users can still override the `background-image` and its embedded `fill` to change its appearance.